src/com/elixr/training/Main.java

```diff
@@ -0,0 +1,72 @@
+package com.elixr.training;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.StringTokenizer;
+import java.lang.String;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+                if(args.length!=2)
+                {
```

Comment:
'{' should be on the same line as if and with space after ')'. This applies for all block like method definition, loops, if statements etc. **Correct this through out the code**.

Reply:
code reformat has been used and changed accordingly